examples/pwa/pwa.html

```diff
@@ -124,6 +124,7 @@
   </div>
 
   <!-- Must be in the very end. -->
+  <script async src="https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/1.0.3/webcomponents-sd-ce.js"></script>
```

Comment:
This file should be forked as `pwa-sd-polyfill.html`

Reply:
`.html`? Did you mean `.js`? I don't think we can use HTML imports

Reply:
I mean the top-level file, `pwa.html` should be cloned and named `pwa-sd-polyfill.html` - this way we can easily test both situations at the same time.

Reply:
Ahhh makes sense, Done

Reply:
But you still need to remove it from here :)